philip/src/components/Post.svelte

```diff
@@ -40,27 +42,27 @@
   let inReply;
   let isReply = false;
 
-  let likes = "n/a";
-  let comments = "n/a";
-  let announces = "n/a";
-
-  if (post.inReplyTo) {
-    inReply = pgi
-      ? post.inReplyTo
-      : ensureObject(post.inReplyTo);
-    isReply = true;
-  }
+  const getCount = async (item, returnAll = false) => {
+    if (!item) return "n/a";
+    const data = typeof item === "string" ? await fetchItem(item) : item;
+    return returnAll ? data : data.totalItems;
```

Comment:
perform fetchItem if pgi, replies should be cached. if not, any way, too much requests, too slow to load.
So, if pgi, and replies is not collection, but url, just display url as string